packages/xstate-fsm/README.md

```diff
@@ -13,7 +13,29 @@
 
 This package contains a minimal, 1kb implementation of [XState](https://github.com/davidkpiano/xstate) for **finite state machines**.
 
-**Features:**
+## Features
+
+|                             | **@xstate/fsm** | [XState](https://github.com/davidkpiano/xstate) |
+| --------------------------- | :-------------: | :---------------------------------------------: |
+| Finite states               |       ✅        |                       ✅                        |
+| Initial state               |       ✅        |                       ✅                        |
+| Transitions (object)        |       ✅        |                       ✅                        |
+| Transitions (string target) |       ✅        |                       ✅                        |
+| Delayed transitions         |       ❌        |                       ✅                        |
+| Eventless transitions       |       ❌        |                       ✅                        |
+| Nested states               |       ❌        |                       ✅                        |
+| Parallel states             |       ❌        |                       ✅                        |
+| History states              |       ❌        |                       ✅                        |
+| Final states                |       ❌        |                       ✅                        |
+| Context                     |       ✅        |                       ✅                        |
+| Entry actions               |       ✅        |                       ✅                        |
+| Exit actions                |       ✅        |                       ✅                        |
+| Transition actions          |       ✅        |                       ✅                        |
+| Parameterized actions       |       ❌        |                       ✅                        |
```

Comment:
Both for this point and Parameterized Guards - 
The official docs don't seem to reference these by this name, is this referring to [Custom Guards](https://xstate.js.org/docs/guides/guards.html#serializing-guards)?

Reply:
I believe this refers to parametrized actions as in:
```
actions: {
  type: 'actionName',
  param: 42.
}
```

That being said - what you refer to (under a custom guards name) is also not supported in `@xstate/fsm` though. Currently, only `actions` can be "provided" through the second argument of the `createMachine` API.